app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -230,6 +201,7 @@ function InPersonLocationStep({ onChange, toPreviousStep }) {
   return (
     <>
       <PageHeading>{t('in_person_proofing.headings.location')}</PageHeading>
+      {arcgisSearchEnabled && <AddressSearch onAddressFound={handleFoundAddress} />}
```

Comment:
As an follow-on to my previous comment and reaffirming why boolean would be better: The string `'false'` is truthy (`!!'false' === true`), which I don't think is the behavior you want.